src/modules/spaces/controllers/spaces.controller.spec.ts

```diff
@@ -168,6 +174,22 @@ describe('SpacesController', () => {
 				error: 'Not Found',
 			});
 		});
+
+		it('should throw a gone exception for space that is inactive', async () => {
+			mockSpacesService.findBySlug.mockResolvedValueOnce(mockSpacesResponse.items[2]);
+
+			let error;
+			try {
+				await spacesController.getSpaceBySlug('huis-van-alijn');
+			} catch (err) {
+				error = err;
+			}
+			expect(error?.response).toEqual({
+				statusCode: 410,
+				message: 'Space with slug "huis-van-alijn" is inactive',
+				error: 'Gone',
+			});
```

Comment:
You could simplify this to:
```suggestion
			try {
				await spacesController.getSpaceBySlug('huis-van-alijn');
				fail(...)
			} catch (err) {
				expect(err?.response).toEqual({
					statusCode: 410,
					message: 'Space with slug "huis-van-alijn" is inactive',
					error: 'Gone',
				});
			}
```